src/interceptors/response.ts

```diff
@@ -15,9 +15,15 @@ export function defaultResponseInterceptor(axios: AxiosCacheInstance): ResponseI
    *
    * Also update the waiting list for this key by rejecting it.
    */
-  const rejectResponse = async (responseId: string, config: CacheRequestConfig) => {
+  const rejectResponse = async (
+    responseId: string,
+    config: CacheRequestConfig,
+    clearCache = true
```

Comment:
```suggestion
    clearCache: boolean
```

Specifically use `true` on other places so its clear that clearing cache is the wanted behavior.